From b2d9a80d241500bb12af7c70e6b790b26d6ccee4 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Tue, 4 Feb 2025 12:00:48 -0800
Subject: [PATCH] remove mypy and flake8 tests

---
 tests/run.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/tests/run.py b/tests/run.py
index 0782723..52939c2 100755
--- a/tests/run.py
+++ b/tests/run.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
 
 import os
 import sys
@@ -7,10 +7,10 @@ import subprocess
 from wasabi import msg
 
 tests = [
-    "mypy --config-file setup.cfg scrubadub_address/",
-    "flake8  --config setup.cfg scrubadub_address/",
+    #"mypy --config-file setup.cfg scrubadub_address/",
+    #"flake8  --config setup.cfg scrubadub_address/",
     # If py3.5 then examples with spacy don't work so disable doctests
-    'if python3 --version | grep -Evq "Python (3\\.5\\.)" ; then nosetests --with-doctest --doctest-extension=rst ./tests/ ./scrubadub_address/ ; else nosetests ; fi',
+    'nosetests --with-doctest --doctest-extension=rst ./tests/ ./scrubadub_address/',
 ]
 
 
-- 
2.39.3 (Apple Git-146)

